tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -129,7 +129,8 @@ func newSession(credentialPath, credentialAccountID, region string) (*awssession
 // resourceTrackerDir.
 // On success, the function outputs RDP access information in the commandline interface. It also returns the
 // the credentials to access the Windows VM created,
-func (a *AwsProvider) CreateWindowsVM() (credentials *types.Credentials, err error) {
+func (a *AwsProvider) CreateWindowsVM() (windowsVM types.WindowsVM, err error) {
```

Comment:
func docs aren't representing the correct parameters

Reply:
Good catch. Updated. PTAL.